src/views/credentials/__tests__/addCredentialModal.test.tsx

```diff
@@ -68,6 +69,29 @@ describe('useCredentialForm', () => {
     }).toMatchSnapshot('token auth');
   });
 
+  it('should derive auth type based on the provided credential', () => {
+    const mockCredential = {
+      id: 1,
+      name: 'Test Credential',
```

Comment:
@cdcabrera I know you don't like to use "complete" mocks in tests, and this mock specifically is problematic because every time we run it again a snapshot will fail because of the created_at, updated_at fields that are dynamic. I thought about adding a hard-coded date, but is there a better way to solve this problem ?